internal/ingress/controller/config/config.go

```diff
@@ -882,6 +882,7 @@ func NewDefault() Configuration {
 			PreserveTrailingSlash:    false,
 			SSLRedirect:              true,
 			CustomHTTPErrors:         []int{},
+			SetProxyInterceptErrors:  true,
```

Comment:
this is the default for ingress annotations only, and not the global config, correct? The global config value should also default to true, shouldn't it?